ruma-client-api/src/r0/sync/sync_events.rs

```diff
@@ -500,4 +502,31 @@ mod tests {
         assert_eq!(req.set_presence, PresenceState::Online);
         assert_eq!(req.timeout, Some(Duration::from_millis(0)));
     }
+
+    #[test]
+    fn timeline_serde() {
+        let timeline = Timeline { limited: true, prev_batch: None, events: vec![] };
+
+        let timeline_serialized = json!({
+            "limited": true,
+            "events": []
+        });
+
+        assert_eq!(to_json_value(timeline).unwrap(), timeline_serialized);
+
+        let timeline_deserialized: Timeline = from_json_value(timeline_serialized).unwrap();
+        assert_eq!(true, timeline_deserialized.limited);
```

Comment:
Can you swap the argument order for the assertions against literals? I don't think we ever do it this way around anywhere else.

Reply:
Done